spring-cloud-kubernetes-fabric8-config/src/test/java/org/springframework/cloud/kubernetes/fabric8/config/KubernetesConfigConfigurationTest.java

```diff
@@ -16,62 +16,92 @@
 
 package org.springframework.cloud.kubernetes.fabric8.config;
 
-import org.junit.Test;
+import io.fabric8.kubernetes.client.KubernetesClient;
+import io.fabric8.kubernetes.client.server.mock.EnableKubernetesMockClient;
+import org.junit.jupiter.api.Test;
+
+import org.springframework.boot.autoconfigure.condition.ConditionalOnMissingBean;
+import org.springframework.context.annotation.Bean;
+import org.springframework.context.annotation.Configuration;
 
 import static org.assertj.core.api.Assertions.assertThat;
 
 /**
  * @author Ryan Dawson
  * @author Kris Iyer - Add tests for #643
  */
+@EnableKubernetesMockClient(crud = true, https = false)
 public class KubernetesConfigConfigurationTest extends KubernetesConfigTestBase {
 
+	// injected because of @EnableKubernetesMockClient. Because of the way
```

Comment:
why this is needed in this manner is explained in a similar class, I hope it makes sense.